isubit/luggage_ckeditor
# LUGG-1229 Update fakeobjects plugin for CKEditor 4.16.2

To test:
* Build a new Luggage site. `cd sites/all/modules/luggage/luggage_ckeditor` and `git checkout LUGG-1299`. Test the Link widget to be sure that it still works as expected.
